CONTINUITY.md

```
@@ -28,29 +29,33 @@ main con el fix de workspace. Fresh install muestra todos los shortcuts.
 ### Ya cerrado
 - ✅ PR #185 — fix install + wizard + addendas
 - ✅ PR #187 — fix departamentos grupo CFDI Recibidos
+- ✅ PR #189 — fix workspace shortcuts + visual
 
 ### En progreso
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)